mindsdb/integrations/handlers/google_search_handler/README.md

```diff
@@ -0,0 +1,82 @@
+# Google Search Console API Integration
+
+This handler integrates with the [Google Search API](https://developers.google.com/webmaster-tools)
+to allow you to use Google Search data in your SQL queries.
+
+## Example: Predicting clicks based on Google Search activity
+
+To see how the Google Search Console handler is used, let's walk through a simple example to create a model to predict
+our clicks on a site based on Google Search activity.
+
+## Connect to the Google Search API
+
+We start by creating a database to connect to the Google Search Console API. Currently, there is no need for an API key:
+
+However, you will need to have a Google account and have enabled the Google Search Console API.
+Also, you will need to have the credentials in a json file. 
+You can find more information on how to do
+this [here](https://developers.google.com/webmaster-tools/v1/prereqs).
+
+**Optional:**  The credentials file can be stored in the google_search handler folder in
+the [mindsdb/integrations/google_search_handler](mindsdb/integrations/handlers/google_search_handler) directory.
+
+~~~~sql
+CREATE
+DATABASE my_Search
+WITH  ENGINE = 'google_search',
+parameters = {
+    'credentials': 'C:\\Users\\panagiotis\\Desktop\\GitHub\\mindsdb\\mindsdb\\integrations\\handlers\\google_search_handler\\credentials.json'
```

Comment:
credentials should be defined as PATH parameter. here is examples from BigQueryHandler:
- [defining](https://github.com/mindsdb/mindsdb/blob/staging/mindsdb/integrations/handlers/bigquery_handler/bigquery_handler.py#L156)
- [using](https://github.com/mindsdb/mindsdb/blob/staging/mindsdb/integrations/handlers/bigquery_handler/bigquery_handler.py#L38)